src/main/java/com/gooddata/util/GoodDataToStringBuilder.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (C) 2004-2016, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.util;
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.builder.RecursiveToStringStyle;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import java.util.Collection;
+
+/**
+ * GoodDataToStringBuilder enables to create string representation based on fields of an object and exclude specified fields
+ * links field is excluded by default.
+ */
+public class GoodDataToStringBuilder extends ReflectionToStringBuilder {
```

Comment:
created new static method `defaultToString(Object, String...)`, which will exclude "links" field by default. Removed override of `toStringExclude(Object, String...)` and `toString(Object, Collection<String>)`.